primitives/session-keys/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2020-2022 Manta Network.
+// This file is part of Manta.
+//
+// Manta is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Manta is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Manta.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Primitives for session keys
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub mod aura;
+pub mod nimbus;
+pub mod vrf;
+pub use vrf::*;
```

Comment:
Why re-export `vrf` keys and not the others?

Reply:
fixed to reexport all 3 pubkey types